deployments/daemon-set/nginx-ingress.yaml

```diff
@@ -48,12 +48,22 @@ spec:
          #  memory: "1Gi"
         securityContext:
           allowPrivilegeEscalation: true
+          readOnlyRootFilesystem: true
           runAsUser: 101 #nginx
           capabilities:
             drop:
             - ALL
             add:
             - NET_BIND_SERVICE
+        volumeMounts:
+        - mountPath: /etc/nginx
+          name: nginx-etc
+        - mountPath: /var/cache/nginx
+          name: nginx-cache
+        - mountPath: /var/lib/nginx
+          name: nginx-lib
+        - mountPath: /var/log/nginx
+          name: nginx-log
```

Comment:
```suggestion
#        volumeMounts:
#        - mountPath: /etc/nginx
#          name: nginx-etc
#        - mountPath: /var/cache/nginx
#          name: nginx-cache
#        - mountPath: /var/lib/nginx
#          name: nginx-lib
#        - mountPath: /var/log/nginx
#          name: nginx-log
```
Volumes will be in the manifest, but commented out to allow users to add them when using `readOnlyRootFilesystem: true`